core/src/main/python/wlsdeploy/util/cla_utils.py

```diff
@@ -102,7 +102,7 @@ class CommandLineArgUtil(object):
     USAGE_ERROR_EXIT_CODE          = 99
     ARG_VALIDATION_ERROR_EXIT_CODE = 98
     PROG_RESTART_REQUIRED          = 103
-    PROG_ROLLBACK_IF_RESTART_EXIT_CODE = 3
+    PROG_ROLLBACK_IF_RESTART_EXIT_CODE = 104
```

Comment:
Just read the update document trying to find the command line argument to rollback. That argument isn't documented and also there are bad RCs in the documentation that need to be removed. 104 needs to be added.

```

    101 - The domain needs to be restarted and the Update Domain Tool needs to be re-invoked with the same arguments.
    102 - The servers impacted by the update operation need to be restarted, in a rolling fashion, starting with the Administration Server, if applicable.
    103 - The entire domain needs to be restarted.
```

Reply:
Added